### DISCUSSION TITLE
how to get peak call result of one cluster

### DISCUSSION THREAD

**Discussion by yingyuan830** [GreenleafLab/ArchR#1546] [Questions / Documentation]
I have checked the documentation and know how to get the merged peak matrix, but I still want to get the peak call result of one specific cluster before the merging, how can I get this information ? Thanks

**Comment by rcorces** [comment]
These are stored in the "PeakCalls" directory of your ArchRProject.

**Reply by urnotchris1** [reply]
In the iterative overlap peak merging procedure, peak calls from multiple cell groups are merged, and I wonder if the peaks in the 'PeakCalls' directory is only the peaks with the highest normalized significance in the reproducibility cluster in that group.   If so, how can I get all the peak calling results for one particular cluster?
Thanks.

**Reply by rcorces** [reply]
I'm not sure I understand your question. Individual pseudobulk replicate peak calls are stored (i.e. all peak calls). Per-group merged calls are only those that pass the merging criteria including significance etc. So we do not produce files that have "all" peak calls per-group. You would have to produce those yourself, in whatever way you think is correct.

**Reply by urnotchris1** [reply]
Peak A has passed the merging criteria and identified in more than one group, but it has the highest significance in cluster ONE.    I wonder if this peak will present in the .rds files of other cell clusters in the "PeakCalls" directory
